codeSnippets/snippets/google-appengine-standard/src/main/appengine/app.yaml

```diff
@@ -1,2 +1,2 @@
-runtime: java11
+runtime: java21
 entrypoint: 'java -jar google-appengine-standard-1.0-SNAPSHOT-all.jar'
```

Comment:
It seems that the entrypoint should also be updated to `java -jar google-appengine-standard-all.jar`

Reply:
You are correct, apologies for overlooking that. I've pushed a fix.